frontend/public/components/monitoring/alertmanager/alertmanager-config.tsx

```diff
@@ -512,21 +519,68 @@ const AlertmanagerConfigWrapper: React.FC<AlertmanagerConfigWrapperProps> = Reac
   },
 );
 
-export const AlertmanagerConfig = () => (
-  <Firehose
-    resources={[
-      {
-        kind: 'Secret',
-        name: 'alertmanager-main',
-        namespace: 'openshift-monitoring',
-        isList: false,
-        prop: 'obj',
-      },
-    ]}
-  >
-    <AlertmanagerConfigWrapper />
-  </Firehose>
-);
+export const AlertmanagerConfig: React.FC = () => {
+  const { t } = useTranslation();
+  const { pathname: url } = useLocation();
+
+  const configPath = '/monitoring/alertmanagerconfig';
+  const YAMLPath = '/monitoring/alertmanageryaml';
+
+  const breadcrumbs = breadcrumbsForGlobalConfig('Alertmanager', configPath);
+
+  return (
+    <>
+      <div className="pf-c-page__main-breadcrumb">
```

Comment:
Not sure why we put the AlertmanagerPage into the config?

Reply:
There used to be two different routers with routes with the exact same paths (alertmanager-page.tsx, which was deleted in the PR, and MonitoringUI from alerting.tsx). Since react-router v6 updated to run using nested links, this caused a problem where the page would not load correctly because of the double routers. I removed the router in alertmanager-page.tsx and updated the Alertmanager config/yaml pages so they would load correctly if visited.